samples/nrf9160/modem_shell/boards/thingy91_nrf9160ns.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# General config
```

Comment:
Is this "General config" necessary? And on the other hand I'm wondering if it would be good to have similar note which is present in asset_tracker_v2:
```suggestion

# Configuration file for Thingy:91.
# This file is merged with prj.conf in the application folder, and options
# set here will take precedence if they are present in both files.
```